src/OpenTelemetry.AutoInstrumentation/Configuration/EnvironmentConfigurationTracerHelper.cs

```diff
@@ -36,7 +36,8 @@ internal static class EnvironmentConfigurationTracerHelper
         [TracerInstrumentation.MySqlData] = builder => builder.AddSource("OpenTelemetry.Instrumentation.MySqlData"),
         [TracerInstrumentation.StackExchangeRedis] = builder => builder.AddSource("OpenTelemetry.Instrumentation.StackExchangeRedis"),
 #endif
-        [TracerInstrumentation.Npgsql] = builder => builder.AddSource("Npgsql")
+        [TracerInstrumentation.Npgsql] = builder => builder.AddSource("Npgsql"),
+        [TracerInstrumentation.GrpcNetClient] = builder => builder.AddGrpcClientInstrumentation(options => options.SuppressDownstreamInstrumentation = !Instrumentation.TracerSettings.EnabledInstrumentations.Contains(TracerInstrumentation.HttpClient))
```

Comment:
I think that this is a good default to use, but it is different than the default used when manually adding the instrumentation library. We should probably add a setting to control this value.

Reply:
I fully agree that it should be configurable, but I would like to avoid making configuration on our side.
It will be great to have it on the library side.

I think that configuration is less important than the instyrumentation itself so I postoped working on this.
When we will more or less ready with tasks for 0.3.0 we should consider focues also on extending instrumentation libraries on sd/contrib repos.

For now, the whole progress in this context is covered by 
 https://github.com/open-telemetry/opentelemetry-dotnet/pull/3457
and the task for configuration on MySql side https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/519.

Reply:
I agree that ideally, the config will be in the instrumentation library implementation. I'm also asking the question to see if we need an interim solution as well, especially if the next release of the instrumentation library or sdk requires S.D.DS. version 7 (which is in prerelease). If we decide that we should have an interim solution, then I'm fine with implementing it in a separate PR; I just want us to make a decision.

Reply:
> I'm also asking the question to see if we need an interim solution

The default position should be to treat it as a limitation of current implementation unless there is a compelling enough case to make our own configuration. I think the case with the gRPC setup (that required an extra setting) was a good example of that.

Reply:
In this specific case I don't think we need an interim solution.

Reply:
>  if the next release of the instrumentation library or sdk requires S.D.DS. version 7 (which is in prerelease)

That's a very good point!